application.go

```diff
@@ -21,22 +21,22 @@ import (
 	"fmt"
 	"net/url"
 	"time"
+	"github.com/golang/glog"
 )
 
 var (
+	// ErrApplicationExists ... the application already exists in the marathon
 	ErrApplicationExists = errors.New("The application already exists in marathon, you must update")
```

Comment:
First off, cheers for the feedback btw ... it is very much welcomed. Yep, i'll removed a error as it not used